src/rgw/rgw_op.cc

```diff
@@ -2872,7 +2872,7 @@ void RGWDeleteBucket::execute()
   }
 
   op_ret = rgw_bucket_sync_user_stats(store, s->user->user_id, s->bucket_info);
-  if ( op_ret < 0) {
```

Comment:
same

Reply:
@mikulely  What about this one? Someone once let me delete the extra spaces.
Are there any guidelines for spaces and blank lines?

Reply:
sometimes whitespace changes do not get picked up while reviewing, otherwise introducing purely whitesepace changes makes it difficult when backporting, or when git blame etc.